verticaltabs.js

```diff
@@ -87,12 +88,33 @@ VerticalTabs.prototype = {
     let window = this.window;
     let document = this.document;
 
-    this.BrowserOpenTab = this.window.BrowserOpenTab;
-    this.window.BrowserOpenTab = function () {
-      this.pushToTop = prefs.opentabstop;
-      this.window.openUILinkIn(this.window.BROWSER_NEW_TAB_URL, 'tab');
-      this.pushToTop = false;
-    }.bind(this);
+    let oldAddTab = window.gBrowser.addTab.bind(window.gBrowser);
+    window.gBrowser.addTab = function (...args) {
```

Comment:
We should re-set `window.gBrowser.addTab` to the previous value in an unload function…